src/Microsoft.SqlTools.ServiceLayer/LanguageServices/LanguageService.cs

```diff
@@ -1506,9 +1565,15 @@ internal SignatureHelp GetSignatureHelp(TextDocumentPosition textDocumentPositio
             // if there are no completions then provide the default list
             if (resultCompletionItems == null)
             {
-                resultCompletionItems = AutoCompleteHelper.GetDefaultCompletionItems(scriptDocumentInfo, useLowerCaseSuggestions);
+                resultCompletionItems = AutoCompleteHelper.GetDefaultCompletionItems(scriptDocumentInfo, useLowerCaseSuggestions);                
```

Comment:
>                 [](start = 130, length = 16)

nit: Please clean up these trailing whitespaces (there's lots of extensions out there that will visualize this for you and/or auto-cleanup)